doc/api/path.md

```diff
@@ -166,12 +166,12 @@ path.extname('.index')
 
 A [`TypeError`][] is thrown if `path` is not a string.
 
-## path.format(pathObject)
+## path.format(options)
```

Comment:
I see your point there with it not being consistent to the code. The reason behind the change was more about having shorter mentions in the description and a (possibly) more consistent user experience when reading the docs. But you're right that it's not actually options.